Taskfile.yml

```diff
@@ -42,38 +42,51 @@ tasks:
       - package.json
       - yarn.lock
 
-  workspaces:build:
-    desc: Build Yarn workspaces
+  workspaces:setenv:
+    desc: Ensure workspace projects have .env files
     vars:
       FLAGS_FILE: '{{.FLAGS_FILE}}'
-    deps: ["workspaces:setup"]
     cmds:
       - |
-        if [[ -n "$FLAGS_FILE" ]]; then
+        if [[ -n "$FLAGS_FILE" ]]
+        then
           echo "Feature flags found ($FLAGS_FILE), copying to packages for build and runtime."
           cp $FLAGS_FILE .env
           cp $FLAGS_FILE workspaces/local-cli/.env
           cp $FLAGS_FILE workspaces/cli-server/.env
           cp $FLAGS_FILE workspaces/cli-scripts/.env
           cp $FLAGS_FILE workspaces/diff-engine/.env
           cp $FLAGS_FILE workspaces/ui-v2/.env.production.local
```

Comment:
i know this isnt part of the PR directly, this should be using something like `$ENVIRONMENT` or the file should be renamed to not include the env

Reply:
hmm there are other considerations like .gitignore etc.; not sure what your intent is

Reply:
i meant the hardcoded "production" in ".env.production.local"

Reply:
Oh  I see - https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used it needs to be production for production builds (from the React build's perspective)

Reply:
ohhhhhh 👍